src/client/camera.h

```diff
@@ -48,6 +48,15 @@ struct Nametag {
 	v3f nametag_pos;
 };
 
+struct CameraModeRestrictions {
+	bool first, third, third_front;
+
+	CameraModeRestrictions()
+	{
+		first = third = third_front = false;
```

Comment:
no, this is C++98 and lesser. This must be done on variable declaration.

```
bool first = false, third = false ...
```

Reply:
Done